nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/credentials/factory/CredentialPropertyDescriptors.java

```diff
@@ -91,4 +91,24 @@ private CredentialPropertyDescriptors() { }
             .sensitive(true)
             .build();
 
+    public static final PropertyDescriptor DELEGATION_STRATEGY = new PropertyDescriptor.Builder()
+            .name("delegation-strategy")
+            .displayName("Delegation Strategy")
```

Comment:
We're trying to get away from `.displayName`, we should only use `.name` for new properties. I think it'd be worth for us to mark `displayName` as deprecated on the API side to make this more obvious.

Reply:
I'd say just remove on the newly added properties and leave the existing ones as-is even if it's not consistent. We did the same in another PR we merged earlier today.